flink-autoscaler/src/main/java/org/apache/flink/autoscaler/JobAutoScalerImpl.java

```diff
@@ -96,6 +97,13 @@ public void scale(Context ctx) throws Exception {
                 return;
             }
 
+            if (!AutoScalerUtils.verifyForbidPeriods(ctx.getConfiguration())) {
```

Comment:
Config validation should be in the validator module, not here.

Reply:
removed to `DefaultValidator`